CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift

```diff
@@ -149,21 +148,34 @@ class OnboardingViewController: UIViewController {
     }
 
     @objc private func nextPage() {
-        let currentPageScreenName = dataSource.pageModels[dataSource.currentPageIndex].analyticsScreen
 
         if dataSource.currentPageIndex < dataSource.pageModels.count - 1 {
             // Next button tapped
-            AnalyticsManager.track(event: .nextStepTapped, screenNameString: currentPageScreenName)
+            track(event: .nextStepTapped, for: dataSource.currentPageIndex)
```

Comment:
[Comment] `dataSource.currentPageIndex` is used 3 times in a row, probably we can declare it 1 time and reuse